pkg/common/series.go

```diff
@@ -0,0 +1,60 @@
+package common
+
+// NOTE: our series only have one value for each timestamp, because multiple values can be expand to single value
+// and I am not sure how InfluxDB and Druid implement multiple values @czheo
+//
+// A multiple value series
+// cpu.0 system=ubuntu, arch=amd64, usage=0.062, idle=0.034, 1412312312
+// Expand to single value series
+// cpu.0.usage system=ubuntu, arch=amd64, 0.062, 1412312312
+// cpu.0.idle  system=ubuntu, arch=amd64, 0.034, 1412312312
+
+// Series is a time series
+type Series struct {
+	// TODO: string or []byte
+	Name      string            `json:"name"`
+	TagKeys   []string          `json:"-"`
+	TagValues []string          `json:"-"`
+	Tags      map[string]string `json:"tag"`
```

Comment:
http://stackoverflow.com/questions/23330781/sort-golang-map-values-by-keys the way you have a sorted map in go is to have an array and keep track the order of keys. Since only debug serialize result can't be deserialize (with easy), I can change the test to. 
````
obj -> serialized -> obj2 && obj == obj2
````
And use `contains` instead of equal in debug serialize test (or just remove it)